tests/unit/states/test_archive.py

```diff
@@ -119,7 +127,7 @@ def test_tar_gnutar(self):
         Tests the call of extraction with gnutar
         '''
         gnutar = MagicMock(return_value='tar (GNU tar)')
-        source = '/tmp/foo.tar.gz'
+        source = os.sep + os.path.join('tmp', 'foo.tar.gz')
```

Comment:
Again, the `os.path.join()` usage seems unnecessary here. Does the test fail without it?